tests/helpers.py

```diff
@@ -50,11 +53,170 @@ def create_resource(
+    )
+    if wait:
+        assert pod_obj.ocp.wait_for_resource(
+            condition=desired_status, resource_name=pod_name
+        ), f"{pod_obj.kind} {pod_name} failed to reach"
+        f"status {desired_status}"
+    return pod_obj
+
+
+def create_secret(interface_type):
+    """
+    Create a secret
+
+    Args:
+        interface_type (str): The type of the interface
+            (e.g. CephBlockPool, CephFileSystem)
+
+    Returns:
+        OCS: An OCS instance for the secret
+    """
+    secret_data = dict()
+    if interface_type == defaults.CEPHBLOCKPOOL:
+        secret_data = defaults.CSI_RBD_SECRET.copy()
+    elif interface_type == defaults.CEPHFILESYSTEM:
+        secret_data = defaults.CSI_CEPHFS_SECRET.copy()
+    secret_data['metadata']['name'] = create_unique_resource_name(
+        'test', 'secret'
+    )
+    del secret_data['data']['kubernetes']
+    secret_data['data']['admin'] = get_admin_key()
+    return create_resource(**secret_data, wait=False)
+
+
+def create_ceph_block_pool():
+    """
+    Create a Ceph block pool
+
+    Returns:
+        OCS: An OCS instance for the Ceph block pool
+    """
+    cbp_data = defaults.CEPHBLOCKPOOL_DICT.copy()
+    cbp_data['metadata']['name'] = create_unique_resource_name(
+        'test', 'cbp'
+    )
+    cbp_obj = create_resource(**cbp_data, wait=False)
+
+    assert verify_block_pool_exists(cbp_obj.name), (
+        f"Block pool {cbp_obj.name} does not exist"
+    )
+    return cbp_obj
+
+
+def create_storage_class(interface_type, interface_name, secret_name):
+    """
+    Create a storage class
+
+    Args:
+        interface_type (str): The type of the interface
+            (e.g. CephBlockPool, CephFileSystem)
+        interface_name (str): The name of the interface
+        secret_name (str): The name of the secret
+
+    Returns:
+        OCS: An OCS instance for the storage class
+    """
+    sc_data = dict()
+    if interface_type == defaults.CEPHBLOCKPOOL:
+        sc_data = defaults.CSI_RBD_STORAGECLASS_DICT.copy()
+        sc_data['parameters']['pool'] = interface_name
+    elif interface_type == defaults.CEPHFILESYSTEM:
+        sc_data = defaults.CEPHFILESYSTEM_DICT.copy()
+
+    mons = (
+        f'rook-ceph-mon-a.{ENV_DATA["cluster_namespace"]}'
+        f'.svc.cluster.local:6789,'
+        f'rook-ceph-mon-b.{ENV_DATA["cluster_namespace"]}.'
+        f'svc.cluster.local:6789,'
+        f'rook-ceph-mon-c.{ENV_DATA["cluster_namespace"]}'
+        f'.svc.cluster.local:6789'
+    )
+    sc_data['metadata']['name'] = create_unique_resource_name(
+        'test', 'storageclass'
+    )
+    sc_data['parameters']['csi.storage.k8s.io/provisioner-secret-name'] = secret_name
+    sc_data['parameters']['csi.storage.k8s.io/node-publish-secret-name'] = secret_name
+    sc_data['parameters']['monitors'] = mons
+    del sc_data['parameters']['userid']
+    return create_resource(**sc_data, wait=False)
+
+
+def create_pvc(interface_type, sc_name):
```

Comment:
I would rather have one method either Utils or Class based and promote that single method, Please check #174

Reply:
I'll move it to pvc resource once https://github.com/red-hat-storage/ocs-ci/pull/174 is merged